.github/workflows/build_dependabot_bundler_pr.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow is based on https://github.com/dependabot/dependabot-actions-workflow
+
+name: Build Dependabot Bundler PR
+on:
+  push:
+    branches:
+      - "dependabot/bundler**"
```

Comment:
There is a bundler branch created? In rook's dependabot PRs I don't see a bundler branch in the name.

Reply:
I think that this is some magic of dependabot, according to their article

Reply:
Interestingly, the dependabot PRs in the rook.github.io repo such as [this](https://github.com/rook/rook.github.io/pull/148) do have the `bundler` branch.